submissions/NadiaVorontsova/dom-api-task/script.js

```diff
@@ -0,0 +1,69 @@
+    }, {
+        title: "Call of Duty: Modern Warfare",
+        image: "assets/img/MW.jpg",
+        text: "Call of Duty: Modern Warfare is a first-person shooter in the Call of Duty series that is a reimagining of the Modern Warfare sub-series. In the game you have to plunge into a brutal and dramatic campaign, speaking on the side of special forces or rebels, fighting in various cities in Europe and the Middle East. In addition to the single story mode, the game features cooperative and session multiplayer."
+    }, {
+        title: "Call of Duty: Black Ops 4",
+        image: "assets/img/CoDBlackOps4.jpg",
+        text: "Call of Duty: Black Ops 4 is a multiplayer first-person shooter that continues the famous Call of Duty series without a story campaign. Instead, the game has introduced a Battle Royale mode called Blackout, in which you can take control of iconic characters from the entire Black Ops sub-series and fight in your favorite locations using iconic weapons. Also, you will find an updated zombie mode, where users need to look for powerful artifacts that are being hunted by some kind of evil."
+    }]
+
+    const listOfItemsElement = document.querySelector('.nav_menu_items');
+    const titleOfSeriesElement = document.getElementById('series_title');
+    const imageOfSeriesElement = document.getElementById('series_image');
+    const textOfSeriesElement = document.getElementById('series_text');
+
+    const burgerButton = document.getElementById('burger_img');
+    const menu = document.getElementById('menu');
+
+    function createMenu(series, parentElement) {
+        series.forEach( ser=> {
+            const li = document.createElement('li');
+            li.textContent = ser.title;
+            parentElement.append(li);
+            li.classList.add('nav_menu_item');
+        })
+    }
+
+    listOfItemsElement.addEventListener('click', ({target}) => {
+        const seriesTitle=target.textContent;
+        const seriesItem=seriesCOD.find((item)=>item.title===seriesTitle);
+        showContent(seriesItem);
+        showMenu();
+    })
+
+    function showContent(data) {
+        titleOfSeriesElement.innerText = data.title;
+        imageOfSeriesElement.src = data.image;
+        imageOfSeriesElement.alt = data.title;
+        textOfSeriesElement.innerText = data.text;
+    }
+
+    function showMenu() {
+        menu.classList.toggle('nav_menu_items_active');
+        burgerButton.classList.toggle('burger_img_active');
+    }
+
+    burgerButton.addEventListener("click", showMenu);
+    createMenu(seriesCOD, listOfItemsElement);
+    showContent(seriesCOD[0]);
```

Comment:
A magic number. Set a variable with an initial number to render.

Reply:
I created const variable for default content.